src/main/resources/liquibase/crop_changelog/17_1_0.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+				   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+	<changeSet author="diego" id="v17.1.0-1">
+		<preConditions onFail="MARK_RAN">
+			<and>
+				<columnExists tableName="stockprop" columnName="rank"></columnExists>
+			</and>
+		</preConditions>
+		<comment>Remove rank column from stockprop table</comment>
+		<dropIndex tableName="stockprop" indexName="stockprop_idx1"></dropIndex>
```

Comment:
According to MySQL documentation, dropping the index should not be required to drop a column. Have you tested that before? See: https://stackoverflow.com/questions/4341897/what-happens-if-i-drop-a-mysql-column-without-dropping-its-index-first 
@cuenyad